packages/core/src/accordion/accordion-content.element.ts

```diff
@@ -33,17 +34,21 @@ import { html, LitElement } from 'lit-element';
  *
  * @beta
  * @element cds-accordion-content
+ * @cssprop --color
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --background
  */
 export class CdsAccordionContent extends LitElement {
   connectedCallback() {
     super.connectedCallback();
     this.setAttribute('slot', 'accordion-content');
   }
   render() {
-    return html`<slot></slot>`;
+    return html`<div class="private-host"><slot></slot></div>`;
   }
 
   static get styles() {
-    return [baseStyles];
+    return [baseStyles, styles];
   }
 }
```

Comment:
To get the scaling and white space working I had to split the styles out to each component (header/content) so the line heights were correct when slotted. Once I moved the styles it was much easier for the theming values to scale properly.

Reply:
Thank you for cleaning this up!